plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -231,14 +231,14 @@ aws.settings.learn_more=Learn more
 aws.settings.sam.location=SAM CLI executable:
 aws.settings.sam.show_all_gutter_icons=Show gutter icons for all potential AWS Lambda handlers
 aws.settings.sam.show_all_gutter_icons_tooltip=When enabled, show gutter icons in source files for all potential Lambda handlers, even if they are not defined in a template or remotely.
-aws.settings.serverless_label=Serverless settings
+aws.settings.serverless_label=Serverless Settings
```

Comment:
while we're here, it would be nice to just get rid of this and put the CLI settings under the "external tools" configurable